idd/Energy+.idd.in

```diff
@@ -62666,12 +62666,15 @@ Controller:WaterCoil,
        \autosizable
   N2 , \field Maximum Actuated Flow
        \type real
+       \minimum 0
+       \autosizable
+       \default autosize
        \units m3/s
        \ip-units gal/min
-       \autosizable
   N3 ; \field Minimum Actuated Flow
        \type real
-       \default 0.0000001
```

Comment:
@Myoldmopar This default has been in the idd as far back as version 1.2.0, so my guess is that it was the idea of the original controller author.